src/Core/Services/Implementations/InMemoryApplicationCacheService.cs

```diff
@@ -30,6 +30,13 @@ public class InMemoryApplicationCacheService : IApplicationCacheService
         return _orgAbilities;
     }
 
+    public async Task<OrganizationAbility> GetOrganizationAbilityAsync(Guid organizationId)
```

Comment:
thought/question (non-blocking): Would marking this explicitly as nullable (`Task<OrganizationAbility?>`) be beneficial? We do so in the authorization handler helpers, so it may be useful to continue to communicate that it may be null to any consumers.

Reply:
Sounds good, thank you!